qa/workunits/suites/fsx.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-git clone git://git.ceph.com/xfstests.git
+git clone https://git.ceph.com/xfstests-dev.git
 cd xfstests
```

Comment:
cd `xfstests-dev`

Reply:
Fixed it. Thanks.

Reply:
@idryomov This change has passed test - can we do your suggested change in another fix pls?

Reply:
The only reason it passed is that `qa/run_xfstests-obsolete.sh` is only used by RBD.  If this was intended to be a CephFS-only PR, then just drop the change to `qa/run_xfstests-obsolete.sh`.  I'll fix it myself (there is likely more to it then just `cd xfstests-dev`) and I'd like to avoid an inter-dependency between PRs when backporting.

Reply:
Fixed it.

Reply:
I was about to merge this but now ptl-tool complains of new remote changes :(

Reply:
What's the `ptl-tool` ?

Reply:
@vshankar You can always use the merge button on Github ;)  However there is a bigger problem:
```
./tasks/restart.py:68:13: F821 undefined name 'os'
```
So it sounds like even your CephFS-only test wasn't complete enough...

Reply:
> What's the `ptl-tool` ?

src/script/ptl-tool.py

Reply:
> So it sounds like even your CephFS-only test wasn't complete enough...

I wonder where is restart.py being used. I do not see the task as part of fs suite run...